### PR TITLE
simpeg v0.21.1

### PR DESCRIPTION
**Pull request by santisoler**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+{% set version = "0.21.1" %}
 
 package:
   name: simpeg
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/S/SimPEG/SimPEG-{{ version }}.tar.gz
-  sha256: 9798031b19c0be04f84a974f66a9595c560d903e2d90bf5587fe3a7c138e1e32
+  sha256: e71c32770a9c3a32b84033d16adacf10bb5357b31dcd7ae5beab7d59f240bf5e
```
Comment (santisoler) [inline recipe/meta.yaml]:
```suggestion
  sha256: 6e7906b23bc6278c667d199e23092d693e165a9a9208dbb92a2bbc7f1a76092b
```

**Comment by santisoler** [comment]
@conda-forge-admin, please rerender

**Comment by santisoler** [comment]
@jcapriot I think this is ready to be merged

**Comment by jcapriot** [comment]
Ideally it would be good to add run constraints for the optional dependancies: https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#run-constrained

**Comment by santisoler** [comment]
Sounds good. I can think about Choclo (`>=0.1.0`) and maybe `fsspec>=0.3.3`. Do we have any constrain on any of the other optional dependencies (Dask, zarr, etc)?

**Comment by jcapriot** [comment]
They are optional (dask primarily), but I don't know of any specific version constraints on them.

**Pull request merged**